Formula/p/purr.rb

```diff
@@ -0,0 +1,26 @@
+class Purr < Formula
+  desc "Versatile zsh CLI tool for viewing and searching through Android logcat output"
+  homepage "https://github.com/google/purr"
+  url "https://github.com/google/purr/archive/refs/tags/2.0.0.tar.gz"
+  sha256 "f4e6c3dcdd386da7d3664fe6c947e9cfdb2e3362ca8c78e47d0445d94d471246"
+  license "Apache-2.0"
+  head "https://github.com/google/purr.git", branch: "main"
+
+  depends_on "make" => :build
+  depends_on "fzf"
+  depends_on "zsh"
+
+  def install
+    system "make", "purr"
+    bin.install "out/purr"
+  end
+
+  def caveats
+    <<~EOS
+      purr requires you to have adb executable in $PATH
+
+      adb is part of Android SDK command line tools. Get them at:
+        https://developer.android.com/studio#command-line-tools-only
```

Comment:
If this tool is for android/adb community, we probably dont need this.

Reply:
You can reference the `android-platform-tools` cask instead of needing to point at the website.

https://github.com/Homebrew/homebrew-core/blob/2f7b86a5ba703776a03505881286d8cf91198ad6/Formula/s/scrcpy.rb#L55-L60

Reply:
Done - thank you for being so helpful!

Reply:
yeah, cask reference might be a better route. 👍